services/bstats/bstats-players.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+
+const Joi = require('joi')
+const schema = Joi.object({
+  players: Joi.number().required(),
+}).required()
+
+module.exports = class BStatsPlayers extends BaseJsonService {
+  static get route() {
+    return {
+      base: 'bstats/players',
+      pattern: ':pluginid',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'players' }
+  }
+
+  async handle({ pluginid }) {
+    const { players } = await this.fetch({ pluginid })
+    return this.constructor.render({ players })
+  }
+
+  async fetch({ pluginid }) {
+    return this._requestJson({
+      schema,
+      url: `https://bstats-api-format.glitch.me/players/${pluginid}`,
+    })
+  }
+
+  static render({ players }) {
+    return {
+      label: 'players',
```

Comment:
You can exclude `label` here (as the value will always be `players` and it is defined in `defaultBadgeData`) and you can also move `color` to `defaultBadgeData` as well